nautobot/docs/release-notes/version-2.3.md

```diff
@@ -0,0 +1,345 @@
+<!-- markdownlint-disable MD024 -->
+
+# Nautobot v2.3
+
+This document describes all new features and changes in Nautobot 2.3.
+
+## Upgrade Actions
+
+### Administrators
```

Comment:
We need to add a stanza message around the change of the unadorned Docker image moving to 3.12 and Apps compatibility. We should recommend pulling a specific python version of the Docker image, at least for upgrading but in general moving forward.

Reply:
See #6082.